src/test/acceptance/features/navigation.feature

```diff
@@ -31,7 +30,6 @@ Feature: Application process navigation is controlled
       | are you pregnant         |
       | card address             |
 
-  @ignore
   Scenario Outline: Navigation is not allowed past the current page in the flow
```

Comment:
Presumably we'll be removing most of the examples in this scenario?

Reply:
Yep, I've left one per navigation page now, that should cover us sufficiently.